contrib/kind.sh

```diff
@@ -150,6 +160,45 @@ print_params
 
 set -euxo pipefail
 
+check_ipv6() {
+  # Collect additional IPv6 data on test environment
+  ERROR_FOUND=false
+  TMPVAR=`sysctl net.ipv6.conf.all.forwarding | awk '{print $3}'`
+  echo "net.ipv6.conf.all.forwarding is equal to $TMPVAR"
+  if [ "$TMPVAR" != 1 ]; then
+    if [ "$KIND_ALLOW_SYSTEM_WRITES" == true ]; then
+      sudo sysctl -w net.ipv6.conf.all.forwarding=1
+    else
+      echo "RUN: 'sudo sysctl -w net.ipv6.conf.all.forwarding=1' to use IPv6."
+      ERROR_FOUND=true
+    fi
+  fi
+  TMPVAR=`sysctl net.ipv6.conf.all.disable_ipv6 | awk '{print $3}'`
+  echo "net.ipv6.conf.all.disable_ipv6 is equal to $TMPVAR"
+  if [ "$TMPVAR" != 0 ]; then
+    if [ "$KIND_ALLOW_SYSTEM_WRITES" == true ]; then
+      sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0
+    else
+      echo "RUN: 'sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0' to use IPv6."
+      ERROR_FOUND=true
+    fi
+  fi
+  if [ "$ERROR_FOUND" == true ]; then 
+    exit 1  
+  fi  
+
+  echo ""
+  echo "Running: Check for /proc/net/if_inet6"
+  [ -f /proc/net/if_inet6 ] && echo '  IPv6 ready system!' || echo '  No IPv6 support found! Compile the kernel!!'
```

Comment:
shouldn't you exit 1 here?

you could use if statements for this and line 196, and continue with the ERROR_FOUND method you were using above.